approval_frame/forms.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import unicode_literals
+
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+from registration.forms import RegistrationFormUniqueEmail
+
+class RegistrationFormCustom(RegistrationFormUniqueEmail):
+    """
+    Subclass of `RegistrationFormUniqueEmail` which includes application-
```

Comment:
Documentation generators are usually clever enough to figure that out from code and the Django code doesn't seem to be uniformly doing it to all classes, so I'm assuming that that's not the case, but I'm not 100% sure as I haven't used Sphinx before. Thanks for bringing it up though, I hadn't heard of Sphinx before :smile: